src/components/browser/BrowseHierarchy.tsx

```diff
@@ -133,8 +118,16 @@ export class BrowseHierarchy extends Component<Props, State> {
       <>
         {asset ? (
           <>
+            {parents &&
```

Comment:
We need some kind of label here for the parents list.

What would you think about differentiating between parent/hierarchy/child divs & cards by background color or an icon?